### PR TITLE
README: always call @configure() in subclasses

### PR DESCRIPTION
**Pull request by odedniv**
TL;DR I fought with this problem for quite a while until figuring it out, this README note would have saved precious time.

In this example:

```coffeescript
class Book extends RailsResource
  @configure url: '/books', name: 'book'

class Encyclopedia extends Book
  @interceptAfterResponse (result) ->
    ...

class StoryBook extends Book
```

The class `StoryBook` will also have the after response interceptor, since `Encyclopedia` added the interceptor to `Book`'s config object.

Calling `@configure` on `StoryBook` will even be worse, making the load order decide whether or not it will have the interceptor.

Calling `@configure()` in every subclass of `RailsResource` (including sub-subclass) before creating any interceptors (or other `@config` changes) will prevent this, since `@configure` copies the current `@config` on every run.

Unless there is a way to call `@configure()` on each extend automatically (of every type), this PR just adds a note to the README.

### CODE REVIEW

**Comment by tpodom** [comment]
Thanks for the update.  I'll have to try and think about this issue some more to see if I can come up with any better way to make this easier.
